prettify_stories.sh

```diff
@@ -0,0 +1,18 @@
+# Add this as a shell script to your WebStorm configurations
+# and as executable command to VS Code
+# Make sure the working directory for this command is equal to "<PROJECT_DIR>/src/stories"
+
+# The path to the prettier config file.
+CONFIG_FILE="../../.pretierrc"
+
+# The path to the prettier ignore file.
+IGNORE_FILE="../../.prettierignore"
+
+# The regex for the exact files to be handled by prettifier.
+# ** = all (sub-)directories in the current working directory.
+# *.stories.tsx = all files which are appended with ".stories.tsx" in the specified directory.
+STORIES_DIR_REGEX="./**/*.stories.tsx"
+
+prettier --config $CONFIG_FILE --ignor-path $IGNORE_FILE all --write $STORIES_DIR_REGEX
```

Comment:
`--ignor-path` :rarr `--ignore-path`

Hehe, I think maybe you should test this script first?